src/app/modules/shared/services/landlord/landlord.service.ts

```diff
@@ -32,10 +32,11 @@ export class LandlordService {
     return of(null)
   }
 
-  delete(id: number): void {
+  delete(id: number): Observable<Landlord[]> {
```

Comment:
Solitamente, un metodo `delete` non ritorna nulla o al massimo ritorna l'elemento rimosso.

Ritornare la nuova lista non ha molto senso in effetti, perche' l'hai modificata in questo modo? Non seguo il ragionamento.

Reply:
Sono d'accordo che per il metodo delete non ha nessuna utilità il ritornare la lista di landlord.
Avevo messo cosi perchè prima di mergiare develop avevo un' altra implementazione e mi è rimasta questa modifica..
Provvedo subito a sistemare.